Cargo.toml

```diff
@@ -84,6 +84,7 @@ indexmap = { version = "1", features = ["serde-1"] }
 lazy_static = "1"
 libflate = "1.2"
 log = { version = "0.4", features = ["kv_unstable"] }
+log4rs = "1"
```

Comment:
I think this dev dependency not a runtime dependency, or did I missed its use outside of tests?